options/components/webBottomContent/WebBottomContent.jsx

```diff
@@ -46,12 +60,17 @@ function WebUrlNewList() {
 }
 
 function HighlightKeyword({ url }) {
-  const { keyword } = useContext(WebSearchContext);
+  const { keyword, userSelectSearchValue } = useContext(WebSearchContext);
 
   return (
-    <div className="mt-3 pt-3 border-t font-normal w-full max-w-[calc(100%-0px)] overflow-hidden text-ellipsis whitespace-nowrap">
-      {url.urlText.includes(keyword) &&
-        url.urlText.split(keyword).map((item, index) => {
+    <>
+      {userSelectSearchValue === "제목 + 키워드 검색" ? (
+        <HighlightTitleKeyword url={url} />
+      ) : (
```

Comment:
```suggestion
    <>
      {userSelectSearchValue === SELECT_VALUE_STATE[2] ? (
        <HighlightTitleKeyword url={url} />
      ) : (
```
해당 부분도 개인적인 의견으로는 상수로 저장하신 SELECT_VALUE_STATE 변수를 활용하는 것도 좋을 것 같습니다!

Reply:
유연한 유지보수를 위해서 좋은 선택인 것 같습니다. 해당 상수 import 시켜서 적용할 수 있도록 하겠습니다.